src/app/navbar/expandable-navbar-section/expandable-navbar-section.component.ts

```diff
@@ -39,11 +39,17 @@ export class ExpandableNavbarSectionComponent extends NavbarSectionComponent imp
    * @param {Event} event The user event that triggered this function
    */
   activateSection(event): void {
+
+    console.log(event.target);
+    
+
     this.windowService.isXsOrSm().pipe(
       first()
     ).subscribe((isMobile) => {
       if (!isMobile) {
+        const targetElement = event.target as HTMLElement;
         super.activateSection(event);
+        targetElement.setAttribute('aria-expanded', 'true'); 
```

Comment:
There are a bunch of failures here from `yarn lint`. It looks like you have trailing spaces on several lines in this file.  Those should be removed until you get `yarn lint` to pass locally.